packages/babel-parser/typings/babel-parser.d.ts

```diff
@@ -219,6 +219,12 @@ interface ParserOptions {
    * AST nodes instead of using the `extra` property.
    */
   createParenthesizedExpressions?: boolean;
+
+  /**
+   * By default, `import(foo)` is parsed as `CallExpression(Import, [Identifier(foo)])`.
+   * Set this to true to parse it as an `ImportExpression` node.
+   */
+  createImportExpression?: boolean;
```

Comment:
Unrelated to the PR: both `Expressions` and `Expression` are used here, maybe we can unify them?

Reply:
Agreed, in that case I tend to name it as `createImportExpressions`, to be consistent with the one of parenthesized expression.